### PR TITLE
More D1 alias syntax fixup

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**
Was missed in the previous PR

### CODE REVIEW

**Comment by jacob-carlborg** [comment]
👌

File: dstep/translator/MacroDefinition.d [hunk 1]
```diff
@@ -2056,7 +2056,12 @@ bool translateFunctAlias(
         if (ident !is null &&
             equal(definition.params, expr.args.map!(a => a.translate(context, params, imports))))
         {
-            output.singleLine("alias %s = %s;", definition.spelling, ident.spelling);
+            version (D1)
+                enum fmt = "alias %2$s %1$s;"
+            else
+                enum fmt = "alias %1$s = %2$s;"
```
Comment (jacob-carlborg) [inline dstep/translator/MacroDefinition.d]: As the build said, missing semicolon.

Comment (mihails-strasuns-sociomantic) [inline dstep/translator/MacroDefinition.d]: Sorry, fixed

**Comment by mihails-strasuns-sociomantic** [comment]
Green

**Pull request merged**